src/events/crawler/scrapers/AUS/NT/index.js

```diff
@@ -0,0 +1,17 @@
+import * as parse from '../../../lib/parse.js';
+import * as fetch from '../../../lib/fetch.js';
+
+const scraper = {
+  country: 'AUS',
+  state: 'Northern Territory',
+  url: 'https://coronavirus.nt.gov.au/',
+  type: 'table',
+  priority: 1,
```

Comment:
Add yourself as a maintainer! I think the source URL is fine here, but could you also add `sources: [{ name: "Whatever this source is called" }]`. Same below